sorc/ncep_post.fd/CALHEL.f

```diff
@@ -44,7 +44,7 @@ SUBROUTINE CALHEL(DEPTH,UST,VST,HELI,USHR1,VSHR1,USHR6,VSHR6)
 !
       use vrbls3d,    only: zmid, uh, vh, u, v, zint
       use vrbls2d,    only: fis, u10, v10
-      use masks,      only: lmv
+      use masks,      only: lmv, gdlat
```

Comment:
@hertneky Can you add a change log in this routine file? Thanks!

Reply:
@WenMeng-NOAA Added thanks.